docs/source/user_guide/benchmarks/physicality.rst

```diff
@@ -74,3 +74,64 @@ Data availability
 -----------------
 
 None required.
+
+
+Diatomics
+=========
+
+Summary
+-------
+
+This benchmark probes the short- to medium-range behaviour of every homonuclear and
+heteronuclear diatomic pair in the periodic table. Each MLIP is evaluated on a 100-point
+linear distance grid spanning 0.18-6.0 Å and the resulting energies and projected forces
+are analysed for unphysical oscillations.
+
+Metrics
+-------
+
+1. Force flips
+
+   Average number of times the projected bond force changes sign. Forces are projected
+   onto the bond axis and values below :math:`10^{-2}` eV/Å are rounded to zero to avoid
+   counting noise-induced flips. A smooth curve should switch from attraction to repulsion
+   only once at the minimum.
+
+
+2. Energy minima
+
+   Mean count of distinct minima in the energy-distance profile. Local minima are
+   found from the second derivative, where a physical diatomic should show a single
+   minimum.
+
+
+3. Energy inflections
+
+   Mean number of inflection points obtained from the second derivative of the energy
+   curve. Inflections are flagged when the second derivative changes sign with a
+   tolerance of 0.5 eV/Å² to avoid counting noise-induced inflections. A physical diatomic
+   curve should show one inflection point.
+
+4. :math:`\\rho(E, \\text{repulsion})`
```

Comment:
This and 5 don't come out quite right. You can build the docs and check any fixes, but I can also look into this tomorrow

<img width="386" height="78" alt="Image" src="https://github.com/user-attachments/assets/cf5f4dc1-41dd-4549-ac1d-4b6333cd712d" />